packages/yoga/src/Menu/web/MenuItem.jsx

```diff
@@ -83,15 +98,18 @@ const MenuItem = forwardRef(
       finalProps.href = href;
     }
 
+    const size = get(theme.yoga.spacing, iconSize, iconSize) || 20;
```

Comment:
Shouldn't `20` be the default value?
```suggestion
const size = get(theme.yoga.spacing, iconSize, 20);
```